applications_tests/gls_navier_stokes_2d/mms2d_cahn_hilliard.output

```diff
@@ -0,0 +1,131 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       256
+   Number of degrees of freedom: 867
+   Volume of triangulation:      4
+   Number of Cahn-Hilliard degrees of freedom: 578
+
+*****************************
+Steady iteration:        1/4
+*****************************
+Newton iteration: 0  - Residual:  0
+  -Tolerance of iterative solver is : 1e-12
+  -Iterative solver took : 0 steps
+		alpha =      1 res = 0
```

Comment:
Unit tests do not contain non-linear and linear solver tolerance since those are generally not portable. Please run the test with verbosity to quiet for both non-linear and linear solvers, otherwise the test will be super fragile everywhere.